utilities/env_librados.md

```diff
@@ -0,0 +1,47 @@
+# Introduce to EnvLibrados
+EnvLibrados is a customized RocksDB Env to use RADOS as the backend file system of RocksDB. It overrides all file system related API of default Env. The easiest way to use it is just like following:
+```c++
+std::string db_name = "test_db";
+std::string config_path = "path/to/ceph/config";
+DB* db;
+Options options;
+options.env = EnvLibrados(db_name, config_path);
+Status s = DB::Open(options, kDBPath, &db);
+...
+```
+Then EnvLibrados will forward all file read/write operation to the RADOS cluster assigned by config_path. Default pool is db_name+"_pool".
+
+# Options for EnvLibrados
+There are some options that users could set for EnvLibrados.
+- buffer_max_size. This variable is the max buffer size for WritableFile. After reaching the buffer_max_size, EnvLibrados will sync buffer content to RADOS, then clear buffer.
+- prefix_pool_map. This option allows user to assign different pool for different directory. Because RocksDB only has 2-level structure (dir_name/file_name), the format of prefix_pool_map key is "/dir_name"(CAN'T be "/dir1/dir2"). All files that don't match any dir in prefix_pool_map will be stored in default pool.
+
+The example of setting options looks like following:
+```c++
+std::map<std::string, std::string> pool_map;
+pool_map["/wal"] = "tmp_wal_pool";               // set pool for WAL files
+pool_map[DEFAULT_POOL_KEY] = "tmp_default_pool"; // set default pool name
```

Comment:
or make this a required argument.  that is probably simplest.

Actaully, since there are only ever 2 or 3 pools, just make 2 arguments...  db_pool, wal_pool should be enough for now.